installer/test/phx_new_test.exs

```diff
@@ -74,7 +74,8 @@ defmodule Mix.Tasks.Phx.NewTest do
                   "<title>Hello PhxBlog!</title>"
 
       # Brunch
-      assert_file "phx_blog/.gitignore", "/node_modules"
+      assert_file "phx_blog/.gitignore", "/node_modules/"
```

Comment:
Should I change those tests to match `/assets/node_modules/`? I would say so.

Reply:
Yeah.

Reply:
Done in https://github.com/phoenixframework/phoenix/pull/2772/commits/61f9544e8322bc03c809678d0dbfe0b3a5a62e39